### ISSUE TITLE
Errors reported during export with version XPlane2Blender v3.4.0-beta1

### ISSUE THREAD

**Issue by quantumac** [der-On/XPlane2Blender#290]
Trying to export my BD-5J using Blender 2.78c which exports just fine under XPlane2Blender v3.2.

Blender file is publicly available and can be found online in the download on this page:

http://forums.x-plane.org/index.php?/files/file/27269-bd-5j-microjet/

The errors reported during export are:

ERROR: Material "wings" in object "Nav_Light_Housing_Right" Normal/Alpha/Specular texture must be "None".
ERROR: Material "wings" in object "Aileron_Base_Right" Normal/Alpha/Specular texture must be "None".
ERROR: Material "wings" in object "Elevator_Trim_Tab" Normal/Alpha/Specular texture must be "None".
ERROR: Material "wings" in object "Main_Wing_Flaps_Base_Left" Normal/Alpha/Specular texture must be "None".
ERROR: Material "wings" in object "Landing_Light_Housing_Right" Normal/Alpha/Specular texture must be "None".
ERROR: Material "wings" in object "Horz_Stabilizer" Normal/Alpha/Specular texture must be "None".
ERROR: Material "wings" in object "Main_Wing_Left" Normal/Alpha/Specular texture must be "None".
ERROR: Material "wings" in object "Main_Wing_Flaps_Left" Normal/Alpha/Specular texture must be "None".
ERROR: Material "wings" in object "Strobe_Light_Housing_Left" Normal/Alpha/Specular texture must be "None".
ERROR: Material "wings" in object "Aileron_Right" Normal/Alpha/Specular texture must be "None".
ERROR: Material "wings" in object "Nav_Light_Housing_Left" Normal/Alpha/Specular texture must be "None".
ERROR: Material "wings" in object "Rudder" Normal/Alpha/Specular texture must be "None".
ERROR: Material "wings" in object "Vert_Stabilizer" Normal/Alpha/Specular texture must be "None".
ERROR: Material "wings" in object "Anti-Servo_Rotation" Normal/Alpha/Specular texture must be "None".
ERROR: Material "wings" in object "Landing_Light_Housing_Left" Normal/Alpha/Specular texture must be "None".
ERROR: Material "wings" in object "Main_Wing_Flaps_Right" Normal/Alpha/Specular texture must be "None".
ERROR: Material "wings" in object "Strobe_Light_Housing_Right" Normal/Alpha/Specular texture must be "None".
ERROR: Material "wings" in object "Main_Wing_Flaps_Base_Right" Normal/Alpha/Specular texture must be "None".
ERROR: Material "wings" in object "Aileron_Base_Left" Normal/Alpha/Specular texture must be "None".
ERROR: Material "wings" in object "Main_Wing_Right" Normal/Alpha/Specular texture must be "None".
ERROR: Material "wings" in object "Aileron_Left" Normal/Alpha/Specular texture must be "None".

**Comment by bsupnik** [comment]
@tngreene  This is almost certainly "authoring error", e.g. the wings material is conflicting with something.  I would like to see you improve the warnings so that the two conflicting materials are named, e.g. something like:

Material "wings" has wings.png as its normal+specular texture, but material "fuselage" has no normal+specular texture.  These materials must use the same texture to be exported to the object "airplane.obj"

In other words, name the exported OBJ and both conflicted materials in one place...

This is not a beta-stopper, e.g. this kind of warning improvement can wait until more serious beta bugs are fixed.  I think this is a case where we provide a better warning in code or provide a better warning by hand forever by answering bug reports about "Why doesn't this work."

**Comment by tngreene** [comment]
Can confirm, this was solved by #294
